src/data/manglendemedvirkning/manglendeMedvirkningTypes.ts

```diff
@@ -5,7 +5,7 @@ export interface NewVurderingRequestDTO {
   vurderingType: VurderingType;
   begrunnelse: string;
   document: DocumentComponentDto[];
-  varselSvarfrist: Date;
+  varselSvarfrist?: Date;
```

Comment:
Klarer vi å tvinge riktig typer her? Kanskje noe sånn her uten at det brekker typingen i post apiet. 
```js
export interface NewVurderingRequestDTO {
  personident: string;
  vurderingType: VurderingType;
  begrunnelse: string;
  document: DocumentComponentDto[];
  varselSvarfrist?: Date;
}

export interface NewForhandsvarselVurderingRequestDTO extends NewVurderingRequestDTO {
  varselSvarfrist: Date;
}
```

Reply:
Hmja, måtte vel definert en egen mutation som krever NewForhandsvarselVurderingRequestDTO som brukes fra forhåndsvarsel-skjema da i så fall. Kan se på det :+1:

Reply:
Ja, vet ikke om det er mulig å få til, men hadde vært kult med noe ish sånn her. Blir jo samme problem i alle de andre vurderingene også 🤔

Reply:
Har forsøkt en variant med forbedrede typer og to mutations nå, én for forhåndsvarsel og én for de andre vurderingene.

Reply:
Synes det ble bra nå 🙌🏼